tests/ha/fencing.pm

```diff
@@ -36,6 +37,12 @@ sub run {
         barrier_wait("HANA_REPLICATE_STATE_${cluster_name}_NODE${node_index}");
     }
 
+    # Modify SELinux configurtion file to take permissive mode effect after rebooting
+    if (get_var("WORKAROUND_BSC1239148") && is_sles4sap()) {
+        record_soft_failure("bsc#1239148: workaround by changing mode to Permissive");
+        assert_script_run("sed -i -r \'s/^SELINUX=.*\$/SELINUX=permissive/g\' /etc/selinux/config");
+    }
```

Comment:
I don't think this is necessary. If `WORKAROUND_BSC1239148` is set, then `hana_install` will set to SELinux to `Permissive` ... but this does give me an idea for a possible change in `check_after_reboot`. Will get back to you.

Reply:
Spoke with Security, and this should be enough to set the system to permissive on reboot. So forget what I suggested about `ha/check_after_reboot`. This can go here in `ha/fencing`, and perhaps even earlier in https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/sles4sap/hana_install.pm#L134-L138